manifests/t/TerraformLinters/tflint/0.44.1/TerraformLinters.tflint.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: TerraformLinters.tflint
+PackageVersion: 0.44.1
+PackageLocale: en-US
+Publisher: Terraform Linters
+PublisherUrl: https://github.com/terraform-linters
+PublisherSupportUrl: https://github.com/terraform-linters/tflint/issues
+# PrivacyUrl:
+Author: Kazuma Watanabe
+PackageName: tfling
```

```suggestion
PackageName: TFLint
```
Looks like a typo; Also we should use the capitalization used by the Publisher here https://github.com/terraform-linters/tflint/#tflint